### ISSUE TITLE
AEUX 0.8.0 Stuck on sending selection from Sketch to Ae

### ISSUE THREAD

**Issue by lostphasma** [google/sketch2ae#104]
**Describe the bug**

**To Reproduce**
1. Go to Sketch Plugin
2. Open Ae Plugin
2. Click on 'Send selection to Ae' from Sketch
3. Stuck on the wheel

**Expected behavior**
Send selection to Ae

I tried with a picture and it works as expected. Nothing happens with boards.

**Desktop (please complete the following information):**
 - OS: macOS 11.6

**Comment by marcusgriffen** [comment]
Same issue. I noticed it only happens with artboards/groups that have images.

**Comment by adamplouff** [comment]
Oops you're absolutely right (trying to juggle too many things). What version of Sketch are you using? I just tested on a couple machines with 62 and 77 with no issues. 

It might not be related but out of curiosity, are your Sketch files local or cloud?

**Comment by jasontcox1** [comment]
Just to add to this, I am not only having this problem since updating to .8.0, but I cant get ANYTHING over to AE. Even the simplest artboards with no images. But it has the same behavior as described above... The AEUX panel in Sketch just spins, and my computer DOES switch over to AE after a couple seconds and then...nothing. Just nothing. Ive rebooted, Ive tried two different versions of AE, different types of artboards. Arrrghhh I depend on this plugin lol.

**Comment by everydayslang** [comment]
Yeah same, images export, but the comp never gets created in AE. I could get a comp created if I removed all the images. This plugin is killer, really. They say you never know a good thing until it's gone... 😢
